docs/guides/integration/github.md

```diff
@@ -239,7 +178,37 @@ steps:
 
 It may improve CI times to store uv's cache across workflow runs.
 
-The cache can be saved and restored with the official GitHub `cache` action:
+The [`astral-sh/setup-uv`](https://github.com/astral-sh/setup-uv) has built-in support for
+persisting the cache:
+
+```yaml title="example.yml"
+- name: Enable caching
+  uses: astral-sh/setup-uv@v1
+  with:
+    enable-cache: true
+```
+
+You can configure the action to use a custom cache directory:
+
+```yaml title="example.yml"
+- name: Define a custom uv cache path
+  uses: astral-sh/setup-uv@v1
+  with:
+    enable-cache: true
+    cache-local-path: "/path/to/cache"
+```
+
+Or invalidate it when the lockfile or its inputs change:
+
+```yaml title="example.yml"
+- name: Define a cache dependency glob
+  uses: astral-sh/setup-uv@v1
+  with:
+    enable-cache: true
+    cache-dependency-glob: "uv.lock"
```

Comment:
I don't know if we support custom `restore-keys` yet, looking into it.

Reply:
Looks like the cache already does this.

Reply:
Yes. You can also supply e.g. `requirements**.txt`